code/dataq_DI1100_Test.py

```diff
@@ -0,0 +1,238 @@
+"""

```

Comment:
Is this file from [here](https://github.com/dataq-instruments/Python/blob/master/binary_comm/DI-1100/DI_1100.py)?

Reply:
Yes it is. Is there a specified way to reference it's origin? or should I just put the link in the file at the start?

Reply:
No, I was just curious! If anything either add a link in the README or create a `documentation` page in the wiki for this repo and post the link to the repo there so we can keep track of the resources for what we use.

Reply:
That sounds like a good idea. I'll add it as a reference to the README then.